src/main/java/org/isf/utils/jobjects/VoDateTextField.java

```diff
@@ -189,38 +197,46 @@ public void focusLost(FocusEvent e) throws IllegalArgumentException {
 			}
 		});
 	}
-	
+
 	/**
 	 * Returns a GregorianCalendar for date use
 	 */
 	public GregorianCalendar getDate() {
 		GregorianCalendar calendar = new GregorianCalendar();
 		calendar.setLenient(false);
```

Comment:
is this ever used now?

Reply:
yes but indirectly.   More analysis needs to be done to see if other related changes are needed.

Reply:
it seems so... ok! Thanks!